test/test_mysql2.rb

```diff
@@ -0,0 +1,39 @@
+require 'test/unit'
+require 'semian/mysql2'
+
+class TestMysql2 < Test::Unit::TestCase
```

Comment:
I'd like to see some more tests, particularly for:

* Toxiproxy slowness tests
* Circuit breaker
* Recovery after circuit break
* Possibly multiple processes

I know that this is painful. The majority of time developing this library will be testing.